db/migrate/20171230125243_add_pages.rb

```diff
@@ -0,0 +1,15 @@
+class AddPages < ActiveRecord::Migration[5.1]
+  def self.up
+    create_table :pages do |t|
```

Comment:
__[Reek]__

[UncommunicativeVariableName] AddPages#self.up has the variable name 't'


<!-- Comment by [SideCI](https://sideci.com) -->